src/services/LocalStorage.ts

```diff
@@ -0,0 +1,17 @@
+import {NotesType} from '../widget/notes/Notes';
```

Comment:
Don't use capital case for services, utils, etc. (LocalStorage.ts -> storage.ts)